packages/theming/atlas/src/native/ts/core/widgets/dropdown.ts

```diff
@@ -91,11 +91,13 @@ export const DropDown: DropDownType = {
         backgroundColor: input.input.backgroundColor
     },
     itemContainer: {
-        // All ViewStyle properties are allowed
+        // All ViewStyle properties & rippleColor & activeOpacity & underlayColor are allowed
         maxWidth: input.itemContainer.maxWidth,
         paddingVertical: input.itemContainer.paddingVertical,
         paddingHorizontal: input.itemContainer.paddingHorizontal,
-        backgroundColor: input.itemContainer.backgroundColor
+        backgroundColor: input.itemContainer.backgroundColor,
+        underlayColor: input.inputContainer.underlayColor,
+        overflow: "hidden"
```

Comment:
I didn't see this line of change in https://github.com/mendix/Atlas-UI-Framework/pull/97/files

Reply:
@mmehmetAliIzci it wasn't needed in Atlas 2 styles